portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -51,4 +53,8 @@ public Date getUploadDate(){
   public long getId(){
     return this.id;
   }
+
+  public String getCurrUserEmail(){
+    return this.currUserEmail;
```

Comment:
Nit: in google java style, we usually avoid abbreviations. Change to currentUserEmail.